eo-maven-plugin/src/test/java/org/eolang/maven/tojos/ForeignTojosTest.java

```diff
@@ -0,0 +1,80 @@
+/*
+ * The MIT License (MIT)
+ *
+ * Copyright (c) 2016-2023 Objectionary.com
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included
+ * in all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NON-INFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+package org.eolang.maven.tojos;
+
+import java.io.IOException;
+import java.nio.file.Path;
+import org.eolang.maven.FakeMaven;
+import org.hamcrest.MatcherAssert;
+import org.hamcrest.Matchers;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.io.TempDir;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+
+/**
+ * Tests from {@link ForeignTojos}.
+ *
+ * @since 0.29.5
+ */
+final class ForeignTojosTest {
+
+    /**
+     * Testable foreign tojos.
+     */
+    private ForeignTojos tojos;
+
+    /**
+     * Set up environment before each test.
+     *
+     * @param tmp Temporary dir.
+     * @todo #2213:60min Create an instance of ForeignTojos in memory.
+     *  It would be better to be able to create an instance of ForeignTojos in
+     *  memory for test purposes.
+     */
+    @BeforeEach
+    void setUp(@TempDir final Path tmp) {
+        this.tojos = new ForeignTojos(() -> new FakeMaven(tmp).foreign());
+    }
+
+    @ParameterizedTest
+    @CsvSource({
+        "abs, abs, true",
+        "org.eolang.int, org.eolang.float, false",
+        "QQ.io.stdout, QQ.io.stdout, true"
+    })
+    void contains(final String name, final String check, final boolean contains) {
```

Comment:
@maxonfjvipon Maybe it's better to create a separate test for the  "not contains" case? 
Actually, what you are doing looks like the anti-pattern "The Free Ride / Piggyback":
https://stackoverflow.com/questions/333682/unit-testing-anti-patterns-catalogue/333883#333883

Reply:
@volodya-lombrozo yeah, thanks, you're right